resources/views/groups/edit.blade.php

```diff
@@ -41,40 +40,67 @@
 </div>
 
 
-<div class="form-group{{ $errors->has('associated_users') ? ' has-error' : '' }}">
-
-    <label for="associated_users[]" class="col-md-3 control-label">
-        {{ trans('general.users') }}
-    </label>
-
-    <div class="col-md-7">
-        <select class="js-data-ajax"
-                data-endpoint="users"
-                data-placeholder="{{ trans('general.select_user') }}"
-                name="associated_users[]"
-                style="width: 100%"
-                id="associated_users[]"
-                aria-label="associated_users[]"  multiple>
-
-                <option value=""  role="option">{{ trans('general.select_user') }}</option>
-                @if(isset($associated_users))
-                    @foreach($associated_users as $associated_user)
-                        <option value="{{ $associated_user->id }}" selected="selected" role="option" aria-selected="true"
-                                role="option">
-                            {{ $associated_user->present()->fullName }} ({{ $associated_user->username }})
-                        </option>
-                    @endforeach
-                @endif
-        </select>
+<fieldset>
+    <x-form-legend help_text="{{ trans('general.add_users_to_group_help') }}">
+        {{ trans('general.add_users_to_group') }}
+    </x-form-legend>
+
+<!-- this is a temp fix for the select2 not working inside modals -->
+
+
+    <div class="form-group">
+        <div class="col-md-12">
+
+        <!-- This hidden input will store the selected user IDs as a comma-separated string to avoid max_input_vars and max_multipart_body_parts php.ini issues -->
+        <input type="hidden" name="users_to_add" id="hidden_ids_box" class="form-control" value="{{ ($associated_users && is_array($associated_users)) ? implode(",", $associated_users->pluck('id')->toArray()) :  '' }}"/>
```

Comment:
This isn't really `users_to_add`, it's actually the full list of members of the group, correct? If so, can we rename it? That name is just a little confusing to me.